packages/babel-helpers/src/helpers-generated.ts

```diff
@@ -348,10 +348,10 @@ const helpers: Record<string, Helper> = {
   ),
-  // size: 593, gzip size: 303
+  // size: 593, gzip size: 305
   set: helper(
     "7.0.0-beta.0",
-    'import superPropBase from"superPropBase";import defineProperty from"defineProperty";function set(e,r,t,o){return set="undefined"!=typeof Reflect&&Reflect.set?Reflect.set:function(e,r,t,o){var f,i=superPropBase(e,r);if(i){if((f=Object.getOwnPropertyDescriptor(i,r)).set)return f.set.call(o,t),!0;if(!f.writable)return!1}if(f=Object.getOwnPropertyDescriptor(o,r)){if(!f.writable)return!1;f.value=t,Object.defineProperty(o,r,f)}else defineProperty(o,r,t);return!0},set(e,r,t,o)}export default function _set(e,r,t,o,f){if(!set(e,r,t,o||e)&&f)throw new TypeError("failed to set property");return t}',
+    'import defineProperty from"defineProperty";import superPropBase from"superPropBase";function set(e,r,t,o){return set="undefined"!=typeof Reflect&&Reflect.set?Reflect.set:function(e,r,t,o){var f,i=superPropBase(e,r);if(i){if((f=Object.getOwnPropertyDescriptor(i,r)).set)return f.set.call(o,t),!0;if(!f.writable)return!1}if(f=Object.getOwnPropertyDescriptor(o,r)){if(!f.writable)return!1;f.value=t,Object.defineProperty(o,r,f)}else defineProperty(o,r,t);return!0},set(e,r,t,o)}export default function _set(e,r,t,o,f){if(!set(e,r,t,o||e)&&f)throw new TypeError("failed to set property");return t}',
```

Comment:
Maybe we should switch the order back 😂 That causes the gzip size to increase by 2 bytes.

Reply:
Note that it doesn't actually matter much, as imports are removed when the helpers are inlined :) 

https://github.com/babel/babel/pull/16501 will stop considering those imports in the computed size